docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json

```diff
@@ -0,0 +1,345 @@
+                          "$id": "#/properties/allocation/properties/group/items/properties/groups",
+                          "type": "array",
+                          "title": "Group Allocation Groups Collection",
+                          "description": "Collection of groups where if the current user is in any of these groups, the variant specified in the group allocation is used.",
+                          "items": {
+                              "type": "string"
+                          }
+                      }
+                  }
+              }
+          },
+          "percentile": {
+              "$id": "#/properties/allocation/properties/percentile",
+              "type": "array",
+              "title": "Percentile Allocation Collection",
+              "description": "A list of objects, each containing a variant name and percentage range for which that variant should be used.",
+              "items": {
+                  "$id": "#/properties/allocation/properties/percentile/items",
+                  "type": "object",
+                  "title": "Percentile Allocation",
+                  "required": [
+                      "variant",
+                      "from",
+                      "to"
+                  ],
+                  "properties": {
+                      "variant": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/variant",
+                          "type": "string",
+                          "title": "Percentile Allocation Variant",
+                          "description": "The name of the variant to use if the calculated percentile for the current user falls in the provided range.",
+                          "pattern": "^(.*)$"
+                      },
+                      "from": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/from",
+                          "type": "number",
+                          "title": "Percentile Allocation From",
+                          "description": "The lower end of the percentage range for which this variant will be used."
+                      },
+                      "to": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/to",
+                          "type": "number",
+                          "title": "Percentile Allocation To",
+                          "description": "The upper end of the percentage range for which this variant will be used."
+                      }
+                  }
+              }
+          },
+          "seed": {
+              "$id": "#/properties/allocation/properties/seed",
+              "type": "string",
+              "title": "Percentile Allocation Seed",
+              "description": "The value which percentile calculations are based on. Calculated percentiles are consistent across features if the user and seed are consistent.",
```

Comment:
"The value, which...." or "The value percentile..."

"Calculated percentile is consistent across features for a given user if the same nonempty seed is used."